### PR TITLE
refactor: changes from youtube to no-cookies

### LINKED ISSUE #1079: Service worker error for preloading of YouTube videos
During loading of youtube videos link via Zubhub project, we encounter a service worker eror.
```javascript
The service worker navigation preload request was cancelled before 'preloadResponse' settled. If you intend to use 'preloadResponse', use waitUntil() or respondWith() to wait for the promise to settle
```

### PR DESCRIPTION
**Pull request by yokwejuste**
## Summary

Description of PR here...
Closes #1079 

## Changes
- The preload link was changed from `https//www.youtube.com` to `https://www.youtube-nocookie.com`
- Changes the theme command in the `make init` for the backend

## References
- [Stackoverflow](https://stackoverflow.com/questions/66099180/react-js-error-the-service-worker-navigation-preload-request-was-cancelled-befo)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 pre-commit-venv
+.idea/
```
Comment (NdibeRaymond) [inline .gitignore]: this is likely a folder that was not generated by this project but rather native to your work environment right? in that case it shouldn't be here. You are better off having this added to your personal `.git/info/exclude` (the syntax is similar to .gitignore). That way this is only found in your local env

File: zubhub_frontend/zubhub/src/components/input/inputScripts.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ export const refactorVideoUrl = url => {
     return url.replace('watch?v=', 'embed/');
   } else {
     if (url.includes('youtu.be')) {
-      return 'https://www.youtube.com/embed/'.concat(url.split('/')[3]);
+      return 'https://www.youtube-nocookie.com/embed/'.concat(url.split('/')[3]);
```
Comment (NdibeRaymond) [inline zubhub_frontend/zubhub/src/components/input/inputScripts.js:8]: you should do the same thing for the first if conditional `if (url.includes('youtube.com')) {`

**Review by NdibeRaymond — CHANGES_REQUESTED** [review]

File: zubhub_frontend/zubhub/src/components/input/inputScripts.js (hunk 2 above)
Comment (yokwejuste) [reply zubhub_frontend/zubhub/src/components/input/inputScripts.js:8]: copied!

File: .gitignore (hunk 1 above)
Comment (yokwejuste) [reply .gitignore]: Thanks for this

**Review by NdibeRaymond — APPROVED** [review]

**Pull request merged**